recipe/meta.yaml

```diff
@@ -1,41 +1,58 @@
 {% set name = "bson" %}
-{% set version = "0.5.9" %}
-{% set sha256 = "84a06dd1fa350d5814d9845abad982e8ac4c9768c4f41afeaa5fdb6d58570158" %}
+{% set version = "0.5.10" %}
+{% set sha256 = "fef2a86682570c3622519ddafb7477e9b876d440eba4e04017b1a5f3b49ccc27" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://github.com/py-bson/bson/archive/refs/tags/{{version}}.tar.gz
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - setuptools
     - pip
+    - wheel
   run:
     - python
     - six
     - python-dateutil >=2.4.0
 
 test:
+  source_files:
+    - bson/tests
   imports:
     - bson
+  requires:
+    - pip
+    - pytest
+  commands:
+    - pip check
+    - pytest -v  # [py<312]
+    # These test files use assertEquals, removed in python 3.12.
+    - pytest -v --ignore="bson/tests/test_array.py" --ignore="bson/tests/test_random_tree.py" # [py>=312]
 
 about:
-  summary: Independent BSON codec for Python that doesn't depend on MongoDB.
   home: https://github.com/py-bson/bson
-  license: BSD-3-Clause
-  license_family: BSD
-  license_file: LICENSE.txt
+  dev_url: https://github.com/py-bson/bson
+  doc_url: https://github.com/py-bson/bson/blob/master/README.rst
+  summary: Independent BSON codec for Python that doesn't depend on MongoDB.
+  description: |
+    Independent BSON codec for Python that doesn't depend on MongoDB. 
+    The bson ObjectId implementation is forked from the PyMongo project,
+    licensed under the Version 2.0 Apache License.
+  license: BSD-3-Clause AND Apache-2.0
```

Comment:
I'm not convinced that this is the appropriate thing to do.

Reply:
Yeah the `AND` is a little off putting, so maybe `OR` instead?

Reply:
Can I ask what your specific concern is? To my knowledge, this is standard procedure for a package with multiple licenses.

Reply:
The license information needs to be accurate. In this case, it's unclear which one is really the license of the project. I'm pretty sure it's not both licenses.

Reply:
Upon further inspection, both licenses are required for the package as a whole. Apache 2.0 is used for `objectid.py` and `py3compat.py`, as they're sourced from `pymongo`, while the rest is under `BSD 3-clause`.

Reply:
> Independent BSON codec for Python that doesn't depend on MongoDB. The bson ObjectId implementation is forked from the PyMongo project, licensed under the Version 2.0 Apache License.

https://github.com/py-bson/bson/tree/0.5.10?tab=readme-ov-file#bson

Reply:
The license of the project is in https://github.com/py-bson/bson/blob/0.5.10/LICENSE however, github is not able to determine which one it is.